benchmarks/benchmark_experiment.py

```diff
@@ -72,6 +72,9 @@ def _is_available(self, experiment_config):
         exclude_tags=()):
       return False
 
+    if cfg_dynamo is not None and self._args.torch_xla2:
```

Comment:
Could you add a comment explaining why this is not available?

Reply:
I didn't see dynamo is supported for torch_xla2. @qihqi to confirm.

Added the comment to clarify here.

Reply:
it hasnt been implemented yet.